content/en/docs/concepts/policy/limit-range.md

```diff
@@ -125,9 +125,9 @@ kubectl get po/busybox1 -n limitrange-demo -o json | jq ".spec.containers[0].res
 ```
 
 - The `busybox-cnt01` Container inside `busybox` Pod defined `requests.cpu=100m` and `requests.memory=100Mi`.
--  `100m <= 500m <= 800m` , The container cpu limit (500m) falls inside the authorized CPU limit range.
--  `99Mi <= 200Mi <= 1Gi` , The container memory limit (200Mi) falls inside the authorized Memory limit range.
--  No request/limits ratio validation for  CPU/Memory , thus the container is valid and created.
```

Comment:
I'm happy with the existing “limit range” wording here.

Reply:
So question on this: I don't see limit range as two separate words in other docs on the website. It always seems to be LimitRange. Why would this be an exception?

Reply:
Here, “limit range” is used to mean “a range of valid values for some subfield(s) of `requests` or `limits`” whereas elsewhere “LimitRange” means “the Kubernetes API object that represents a constraint on acceptable values for some subfield(s) of `requests` or `limits`” .

The name LimitRange is kind of unhelpful because it implies it's a constraint specifically on `limits`, rather than being applicable to both `requests` and `limits`.

Reply:
Ah, tricky! I agree, and it makes sense. I'll change back this and the others you have highlighted. Thank you.